apps/src/music/player/MusicPlayer.ts

```diff
@@ -46,6 +46,14 @@ interface SoundEvent extends PlaybackEvent {
   effects?: Effects;
 }
 
+interface PatternEvent extends PlaybackEvent {
+  type: 'pattern';
+  id: 'pattern';
+  value: any;
```

Comment:
Can we type this? Something like
```
value: {
  kit: string;
  events: TickEvent[];
}
...

interface TickEvent (or another name) {
   tick: number;
   src: string;
}
```

Edit: could even then remove `value` as a key and have `kit` and `tickEvents` exist on the `PatternEvent` interface directly

Reply:
Good idea.  I have kept them in this `value` member for now, like in your code sample, because I suspect we might want `PatternEvent` to have some variants soon, e.g. arpeggiator, note sequences, etc.